Base/QTGUI/qSlicerScriptedLoadableModuleFactory.cxx

```diff
@@ -77,6 +61,22 @@ qSlicerAbstractCoreModule* ctkFactoryScriptedItem::instanciator()
     }
 #endif
 
+  return true;
+}
+
+//----------------------------------------------------------------------------
+qSlicerAbstractCoreModule* ctkFactoryScriptedItem::instanciator()
```

Comment:
It would make sense, that said we would have to rename the function in CTK .. https://github.com/commontk/CTK/search?utf8=%E2%9C%93&q=instanciator&type=

This spelling originate from [french](https://fr.wiktionary.org/wiki/instanciation) ... 

To avoid breaking other project, I would prefer not to change it.